libcontainer/cgroups/fs/blkio.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
```

Comment:
NIT: I'd have preferred that the new package be called "utils", but that's not a big deal.

Reply:
I avoided `utils` because it is likely to be a random set of mutually unrelated functions.